docs/start/install.rst

```diff
@@ -73,8 +75,10 @@ This step is useful when you want to make modification or obtain a specific vers
         python=3.11
```

Comment:
```suggestion
        python=3.11 \
        ninja
```

I don't understand why the original instructions combined `conda` and `pip`, but since you're removing the latter part, the dependencies should probably be installed via `conda` instead (in fact, I dare say they should have been like this since the beginning).

Technically, you could also include `pybind11` and `pytorch` here, and use `--no-build-isolation` in pip, then you'd be able to build against GPU-enabled PyTorch from `conda-forge`.